docs/content/using/changing_node_connectivity.md

```diff
@@ -0,0 +1,18 @@
+# Changing node connectivity
+
+Node connectivity for dynamic host definition needs to be updated when node connectivity changes take place.
+
+Before you begin, ensure that the old host definition is deleted.
```

Comment:
this is not mandatory if `allowDelete` is set to `true`

Reply:
updated wording accordingly

Reply:
I don't see any change here. please verify the diff **on github**, **before** confirming